packages/@sanity/code-input/src/config.ts

```diff
@@ -2,20 +2,27 @@ import {CodeInputLanguage} from './types'
 
 export const SUPPORTED_LANGUAGES: CodeInputLanguage[] = [
   {title: 'Batch file', value: 'batchfile'},
+  {title: 'C#', value: 'csharp'},
   {title: 'CSS', value: 'css'},
-  {title: 'SASS', value: 'sass'},
-  {title: 'SCSS', value: 'scss'},
+  {title: 'GoLang', value: 'golang'},
```

Comment:
```suggestion
  {title: 'Go', value: 'golang'},
```
I think Go is usually just referred to as "Go" unless you need to differentiate between the language and the.. game? word?

Reply:
Fixed.